tsconfig.json

```diff
@@ -1,9 +1,6 @@
 {
-  "extends": "./tsconfig.base",
-  "include": [
-    "src/**/*"
-  ],
-  "exclude": [
-    "src/visGeometry/workers/**/*"
-  ]
+    "extends": "./tsconfig.base",
+    "include": ["src/**/*", "vitest.config.ts"],
+    "exclude": ["src/visGeometry/workers/**/*"],
+    "compilerOptions": {"types": ["vitest/globals"]}
```

Comment:
this line that predefines the vitest globals probably should only be enabled when we are running tests (separate tsconfig for tests)

Reply:
probably should just make a ticket for it and see if we can do it soon-ish.    I would hate to have weird typechecks passing only because we accidentally enabled extra types.